File: sample-conference-videochat/sample-conference-videochat/Components/ChatScreen/Views/ChatCollectionView/ChatAttachmentOutgoingCell/ChatAttachmentOutgoingCell.m
//
//  ChatAttachmentOutgoingCell.m
//  sample-conference-videochat
//
//  Created by Injoit on 2/25/19.
//  Copyright © 2019 Quickblox. All rights reserved.
//

#import "ChatAttachmentOutgoingCell.h"
#import "UIView+Chat.h"

@implementation ChatAttachmentOutgoingCell

- (void)awakeFromNib {
    [super awakeFromNib];

    [self.previewContainer  roundCornersWithRadius:6.0f isIncoming:NO];
}

@end

File: sample-conference-videochat-swift/sample-conference-videochat-swift/Components/ChatScreen/Chat/CollectionView/CollectionCells/ChatAttachmentOutgoingCell/ChatAttachmentOutgoingCell.swift
//
//  ChatAttachmentOutgoingCell.swift
//  Swift-ChatViewController
//
//  Created by Injoit on 1/28/19.
//  Copyright © 2019 Quickblox. All rights reserved.
//

import UIKit

class ChatAttachmentOutgoingCell: ChatAttachmentCell {
    override func awakeFromNib() {
        super.awakeFromNib()

        previewContainer.roundCorners(radius: 6, isIncoming: false)
    }
}
